plugins/com.aptana.jira.core/src/com/aptana/jira/core/JiraManager.java

```diff
@@ -276,15 +278,14 @@ public JiraIssue createIssue(JiraIssueType type, JiraIssueSeverity severity, Str
 			{
 				severityJSON = severity.getParameterValue() + ",\n"; //$NON-NLS-1$
 			}
-
-			// If we're submitting against TC, we can't do version, we need to stuff that into the Environment
-			if (TITANIUM_COMMUNITY.equals(projectKey))
+			String projectVersion = getProjectVersion();
```

Comment:
@pinnamur It's already running in the job, so it won't cause any problem.
https://github.com/aptana/studio3/blob/development/plugins/com.aptana.jira.ui/src/com/aptana/jira/ui/handlers/SubmitTicketHandler.java#L62